examples/src/test/kotlin/ConnectTest.kt

```diff
@@ -63,7 +63,7 @@ internal class ConnectionTest {
         }
         // :snippet-end:
         higherScopedClient = mongoClient
-        assertEquals(1, higherScopedCommandResult["ok"])
+        assertEquals(1.0, higherScopedCommandResult["ok"].toString().toDouble())
```

Comment:
Fixes this test so it always passes (when running locally, actual is `1.0`, but when running through the GH action, actual is `1`).

Reply:
(The other random test file updates were to trigger the test)